### ISSUE TITLE
Some subcommand descriptions are missing

### ISSUE THREAD

**Issue by szepeviktor** [humanmade/orphan-command#6]
![image](https://github.com/humanmade/orphan-command/assets/952007/f4560da6-0d27-43d5-9e8b-48aa63e30af7)

@johnbillion Would you merge a PR fixing these?

**Comment by szepeviktor** [comment]
Or they must have a sub-subcommand specified that is why they don't have a description?

**Comment by roborourke** [comment]
@szepeviktor yeah - because `wp orphan blog` isn't a command on its own there's nothing for WP CLI to show unfortunately. Would require a bit of reworking to solve, or just rename the meta commands where there's no parent command to `wp orphan blog-meta` etc... for example. Or flip it to `wp orphan meta blog` etc. I think the first option is probably best longer term, but will need to be a breaking change release.

If you want to have a go at it I'll keep an eye on this repo and any PRs.
